src/main/java/database/JPA/AccountDAO.java

```diff
@@ -0,0 +1,62 @@
+package database.JPA;
+
+import java.util.List;
+
+import javax.persistence.EntityManager;
+import javax.persistence.TypedQuery;
+import javax.transaction.Transactional;
+
+import entities.Account;
+
+
+public class AccountDAO extends DAO{
+		//edit or save the account into the database
```

Comment:
mover esse comentario para junto do comentario abaixo. 
/*
 * Edit or save the account into the database
 *  ...